addons/pos_payconiq/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -0,0 +1,73 @@
+import { PaymentScreen } from "@point_of_sale/app/screens/payment_screen/payment_screen";
+import { patch } from "@web/core/utils/patch";
+import { _t } from "@web/core/l10n/translation";
+
+patch(PaymentScreen.prototype, {
+    async setup() {
+        await super.setup(...arguments);
+        this.orm = this.env.services.orm;
+        this.bus_service = this.env.services.bus_service;
+
+        if (!this.pos._hasPayconiqBusHandler) {
+            this.bus_service.subscribe("pos_sync_payconiq", this._onPayconiqSync.bind(this));
```

Comment:
You can use `onNotified` instead.

```js
// point_of_sale/static/src/utils.js
export function getOnNotified(bus, channel) {
    bus.addChannel(channel);
    return (notif, callback) => {
        bus.subscribe(`${channel}-${notif}`, callback);
        return bus.unsubscribe(`${channel}-${notif}`, callback);
    };
}

// pos_payconiq/static/src/app/screens/payment_screen/payment_screen.js
onMounted(() => {
    this.payconiqWebsocket = this.pos.data.onNotified(
        "POS_SYNC_PAYCONIQ",
        this._onPayconiqSync.bind(this)
    );
});

onWillUnmount(() => {
    this.payconiqWebsocket && this.payconiqWebsocket();
});
```

From the backend you'll need to send the message from the pos_config with _notify